src/pal/src/exception/seh-unwind.cpp

```diff
@@ -107,12 +107,16 @@ static void UnwindContextToWinContext(unw_cursor_t *cursor, CONTEXT *winContext)
 
 static void GetContextPointer(unw_cursor_t *cursor, int reg, PDWORD64 *contextPointer)
 {
+#if defined(__APPLE__)
+    unw_get_reg(cursor, reg, (unw_word_t *) contextPointer);
```

Comment:
This doesn't do anything useful, right? It just sets the contextPointer value to the value of the register. I understand there is no API in the libunwind on OSX for that. I will write a standalone comment on the issue and possible ways to solve it.